backend/src/users/user-achievements/user-achievements.controller.ts

```diff
@@ -32,11 +32,16 @@ export class UserAchievementsController {
 		}
 	}
 
-	@Get()
-	async findAll() {
-		return this.userAchievementsService.findAll({
-			relations: { users: true, achievements: true },
-		});
+	// @Get()
+	// async findAll() {
+	// 	return this.userAchievementsService.findAll({
+	// 		relations: { users: true, achievements: true },
+	// 	});
+	// }
+
+	@Get(':id')
+	async findAll(@Param('id') id: number) {
+		return this.userAchievementsService.findAll({ where: { user_id: id } });
```

Comment:
this now does not really do "all user-achievements" anymore, right?
it does "all user-achievements of user with id = id"

so maybe leaving it here it was
and doing this
- user controller `@Get(':id/achievements')` (to do what you do here, get achievements of a single user)
- me controller `@Get('achievements') (to get achievements of current user)